src/renderer/helpers/api/local.js

```diff
@@ -826,7 +828,9 @@ export function parseLocalListVideo(item) {
       lengthSeconds: isNaN(video.duration.seconds) ? '' : video.duration.seconds,
       liveNow: video.is_live,
       isUpcoming: video.is_upcoming || video.is_premiere,
-      premiereDate: video.upcoming
+      premiereDate: video.upcoming,
+      is4k: video.is_4k,
+      hasCaptions: video.has_captions
```

Comment:
(Not line specific or even PR specific)
This is why I dislike missing trailing commas in general - the extra diff and more difficult to duplicate lines
Just a rant not request to change
![image](https://github.com/FreeTubeApp/FreeTube/assets/1018543/4d1ea72b-c8f2-4aac-b0e2-01aaaca2eb3a)